src/routes.js

```diff
@@ -55,4 +63,37 @@ export default async(fastify, _) => {
         }
         return response.code(404).type('text/plain').send('file not found');
     });
+
+    // Upload video page
+    fastify.get('/uploadpage', async(request, response) => {
```

Comment:
I'm thinking we might want the html file to sit in the `site` dir and be served from the a `*.html` endpoint, like our js file(s). What do you think?

Reply:
i'm not sure we will really have endpoint of 'uploadpage'. this was just for the poc. it is temporary anyway.

Reply:
We will serve the upload UI from somewhere. But we might do all the UIs from a single html 🤔